src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const personPartnerSex = sex === 'm' ? 'wife' : 'husband';
+  const personPartner = isMarried
+    ? `${partnerName} is my ${personPartnerSex}`
+    : 'I am not married';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">{personPartner}</p>
```

Comment:
```suggestion
      <h2 className="Person__name">{`My name is ${name}`}</h2>
      
      {age && <p className="Person__age">{`I am ${age}`}</p>}
      
      <p className="Person__partner">{personPartner}</p>
```
Use same code style like in HTML, separate adjacent elements.